README.md

```diff
@@ -54,7 +68,8 @@ where the Conjur object is logged in to the account & ready for use.
 
 ### Variable Operations
 
-A Variable is an access-controlled list of encrypted data values. The values in a Variable are colloquially known as ?secrets?.
+A Variable is an access-controlled list of encrypted data values. The values in a Variable are colloquially known as
```

Comment:
This isn't quiet right.  A variable is a named reference.  The most common thing a variable references is a secret. A variable can have one or more (up to 20 secrets) associated with it, and ordered in reverse chronological order.